mind_your_stonks/bet_client.py

```diff
@@ -132,17 +150,23 @@ def _get_number_of_pages_for_table(self):
 
     def compute_money_invested(self):
         money_invested = 0.00
-        num_of_pages = self._get_number_of_pages_for_table()
-        # Get the table object
-        table = self.driver.find_element_by_class_name("stdTable")
-        for page in range(1, num_of_pages+1):
-            if page > 1:
+        number_of_pages = self._get_number_of_pages_for_table()
+        first_page = 1
+        last_page = number_of_pages + 1
+
+        for page_number in range(first_page, last_page):
+            self.web_setup.logger.info(f"Computing stakes on page: {page_number}.")
+            if page_number > first_page:
                 # Need to get the pagination element again or else raises
                 # StaleElementReferenceException
                 pagination = self.driver.find_element_by_class_name("pagination")
-                page_ = pagination.find_element_by_link_text('{}'.format(page))
-                page_.click()
+                page = pagination.find_element_by_link_text(f"{page_number}")
+                page.click()
+                WebDriverWait(self.driver, TIMEOUT).unitl(
+                    condition.visibility_of_element_located(By.CLASS_NAME, "stdTable"))
```

Comment:
Consider changing this to `condition.presence_of_element_located` instead. Should be sufficient to get the job done.